actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -258,6 +260,18 @@ def test_with_fragment_cache
     @controller.config.perform_caching = true
   end
 
+  def test_with_fragment_cache_when_log_disabled
+    @controller.config.perform_caching = true
```

Comment:
Is the config going to be respected at this time? Won't it have been moved to another attribute by now?